docs/man_page_template.jinja

```diff
@@ -0,0 +1,134 @@
+.TH scap-security-guide 8 "26 Jan 2013" "version 1"
+
+.SH NAME
+SCAP Security Guide - Delivers security guidance, baselines, and
+associated validation mechanisms utilizing the Security Content
+Automation Protocol (SCAP).
+
+
+.SH DESCRIPTION
+The project provides practical security hardening advice for Red Hat products,
+and also links it to compliance requirements in order to ease deployment
+activities, such as certification and accreditation. These include requirements
+in the U.S. government (Federal, Defense, and Intelligence Community) as well
+as of the financial services and health care industries. For example,
+high-level and widely-accepted policies such as NIST 800-53 provides prose
+stating that System Administrators must audit "privileged user actions," but do
+not define what "privileged actions" are. The SSG bridges the gap between
+generalized policy requirements and specific implementation guidance, in SCAP
+formats to support automation whenever possible.
+
+The projects homepage is located at:
+https://www.open-scap.org/security-policies/scap-security-guide
+
+{{{ profiles }}}
```

Comment:
Instead of just concatenating profiles here consider using jinja2's `for`. See http://jinja.pocoo.org/docs/2.10/templates/

That way you can keep the formatting in the actual template.

Reply:
That sounds great, I'll look into this possibility.

Reply:
That sounds great, I'll look into this possibility.

Reply:
That sounds great, I'll look into this possibility.

Reply:
That sounds great, I'll look into this possibility.

Reply:
That sounds great, I'll look into this possibility.